pkg/sbproxy/reconcile/reconcile_visibilities_test.go

```diff
@@ -155,6 +157,10 @@ var _ = Describe("Reconcile visibilities", func() {
 		}, nil)
 	}
 
+	brokerProxyName := func(brokerName, brokerID string) string {
+		return fmt.Sprintf("%s%s-%s", reconcile.DefaultProxyBrokerPrefix, brokerName, brokerID)
```

Comment:
Instead of defining this function both in the reconcile_visibilities and reconcile_broker test files you can define it in `reconcile_suite_test.go` and reuse it from there. Same thing applies for the notification tests